src/forwarder.spec.ts

```diff
@@ -35,6 +36,59 @@ function awsMock(existingSubs: { [key: string]: any }, stackName?: string): Aws
   } as Aws;
 }
 
+describe("canSubscribeLogGroup",() => {
+  it("tells addCloudWatchForwarderSubscriptions function it can subscribe to a given log group with 0 existing subscription filters", async() => {
+
+   const aws = awsMock({});
+   const logGroupName: string = "/aws/lambda/serverless-plugin-test-dev-hello";
+   const expectedSubName: string = "serverless-plugin-test-dev-HelloLogGroupSubscription";
+
+   const canSubscribe = await canSubscribeLogGroup(aws,logGroupName,expectedSubName);
+   expect(canSubscribe).toBeTruthy();
```

Comment:
Instead of `.toBeTruthy()` and `.toBeFalsy()` I would use `.toBe(true)` and `.toBe(false)`.

Truthy means that the variable will evaluate to true in an if statement. So the following values are all truthy: `true`, `754`, and `potato`. Likewise falsy could include `null` or `undefined`. We really want to check that the function actually returns `true`, not just something "truthy".

Reply:
That is great to know thanks!